actionmailer/lib/action_mailer/delivery_job.rb

```diff
@@ -13,7 +13,9 @@ class DeliveryJob < ActiveJob::Base # :nodoc:
     rescue_from StandardError, with: :handle_exception_with_mailer_class
 
     def perform(mailer, mail_method, delivery_method, *args) #:nodoc:
-      mailer.constantize.public_send(mail_method, *args).send(delivery_method)
+      params = args.first if args.first.is_a?(Hash)
```

Comment:
Would not this use a parameterize mailer for a regular mailer that has as first argument a hash?